pkgdb/Makefile

```diff
@@ -146,8 +142,17 @@ endif  # ifndef TOOLCHAIN
 
 # ---------------------------------------------------------------------------- #
 
+# Read info from files
+# --------------------
+
 VERSION := $(file < $(PKGDB_ROOT)/version)
 
+# Files used by `pkgdb buildenv' need to be added to the `nix' store.
+PROFILE_D_SCRIPT_DIR ?= $(shell                         \
+  $(NIX) store add-path -n profile.d src/buildenv/etc)
```

Comment:
nonblocking: iss the fallback for development? Could we fall back to the actual file then?

Reply:
This is set as a fallback.
The path must be a `nix` store path otherwise it'll throw an error AFAIK.